utils/src/main/kotlin/OrtAuthenticator.kt

```diff
@@ -73,6 +73,8 @@ class OrtAuthenticator(private val original: Authenticator? = null) : Authentica
         }
     }
 
+    private val serverAuthentication = mutableMapOf<String, PasswordAuthentication>()
```

Comment:
It now keeps the plain-text password in memory. I guess it is not a problem as the process seems to have read access to the netrc file anyway. What do you think?

Reply:
It's probably not super-nice, but the passwords were kept in memory already before as part of the PasswordAuthentication instances which are passed as return values. Also proxy passwords are already cached in a similar manner.